frc46_token/src/token/state.rs

```diff
@@ -509,18 +511,24 @@ impl TokenState {
         let mut balance_sum = TokenAmount::zero();
         let mut maybe_err: Option<StateInvariantError> = None;
         let balances = self.get_balance_map(bs)?;
-        let res = balances.for_each(|owner, balance| {
+        let res = balances.for_each(|owner_key, balance| {
+            let owner = match u64::decode_var(owner.0.as_slice()) {
```

Comment:
Could you pull this decode out to a nicely-named function alongside actor_id_key?

Reply:
Done!